core/src/apps/cardano/layout.py

```diff
@@ -421,6 +428,14 @@ async def show_warning_address_foreign_staking_key(
     )
 
 
+async def show_warning_tx_network_unverifiable(ctx: wire.Context) -> None:
+    await show_warning(
+        ctx,
+        ("Transaction has no", "outputs, network", "cannot be verified"),
+        button="Ok",
```

Comment:
this is pointless if the user cannot refuse to proceed :eyes:

this should be a "continue? yes/no" dialog

Reply:
fair point - the original rationale was that this is meant to be a warning and the warning screen seemed to visually convey that quite nicely, but it makes sense in this case to have the option to reject it in a less abrupt way than completely unplugging the device (such as the "hold to confirm" prompt) as it conveys an information that the user should decide on.

It's worth pointing out that in the Cardano app already uses the `show_warning` in similar scenarios - see e.g. the `show_warning_address_foreign_staking_key` function above, but I agree it should be refactored for the sake of consistency though I would rather postpone those for another PR as they concern rather obscure edge cases and would bloat this PR. But I'd do this one (arguably the likeliest to happen) with the yes/no prompt.